codespell_lib/data/dictionary.txt

```diff
@@ -12368,6 +12406,7 @@ recurively->recursively
 recurrance->recurrence
 recursivion->recursion
 recursivly->recursively
+recuse->recuse, recurse,
```

Comment:
this will return false positives but that's OK IMO

Reply:
@luzpaz Agreed.  I found several occurrences of `recuse` in the codebases I was correcting, and they were all misspellings of recurse.  I was tempted to make it the first option since `recurse` is used much more frequently in code, but decided since it is a real word to leave it as the first option.  If you think they should be switched, PLMK.

Reply:
Again, personally I don't think we should have it in the main file